### PR TITLE
SLI-1102 Emphasize focus on new code in the 'Taint Vulnerabilities' tab

### PR DESCRIPTION
**Pull request by nquinquenel**

### CODE REVIEW

**Review by eray-felek-sonarsource — APPROVED** [review]
LGTM!

File: src/main/java/org/sonarlint/intellij/ui/nodes/SummaryNode.java [hunk 1]
```diff
@@ -64,13 +66,18 @@ public String getText() {
       return emptyText;
     }
 
-    if (forSecurityHotspot) {
-      return String.format(FORMAT, findings, findings == 1 ? "Security Hotspot" : "Security Hotspots", files, files == 1 ? "file" : "files");
+    String sinceText = "";
+    String newOrOldOrNothing = "";
```
Comment (eray-felek-sonarsource) [inline src/main/java/org/sonarlint/intellij/ui/nodes/SummaryNode.java:70]: Maybe we can have a better name here, how about focusText?

**Review by eray-felek-sonarsource — APPROVED** [review]
Small comment other than that LGTM!

File: src/main/java/org/sonarlint/intellij/ui/nodes/SummaryNode.java (hunk 1 above)
Comment (damien-urruty-sonarsource) [reply src/main/java/org/sonarlint/intellij/ui/nodes/SummaryNode.java:70]: Not sure the suggestion is clearer. Naming is hard... Let's revisit later

**Pull request merged**